delivery_postlogistics/models/stock_quant_package.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2013-2016 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, api, exceptions, fields, models
+
+
+class StockQuantPackage(models.Model):
+    _inherit = "stock.quant.package"
+
+    postlogistics_manual_cod_amount = fields.Float(
+        string="Postlogistics Cash On Delivery Amount",
+        help="If the cash on delivery amount for this package is different "
+        "than the total of the sales order, write the amount there.",
+    )
+    parcel_tracking = fields.Char("Parcel Tracking")
+    package_carrier_type = fields.Selection(related="packaging_id.package_carrier_type")
+
+    @api.returns("stock.picking")
+    def _get_origin_pickings(self):
+        self.ensure_one()
+        operation_model = self.env["stock.pack.operation"]
```

Comment:
hmm am I wrong or this model does not exists in v13? Shouldn't you look for s.move.line? :thinking: 
If this is true, it means this piece of code is not called by any test. If so, please fix test cov too. :pray:

Reply:
updated